packages/gatsby-source-contentful/src/normalize.js

```diff
@@ -1,10 +1,18 @@
 // @ts-check
 import stringify from "json-stringify-safe"
 import _ from "lodash"
+import { getGatsbyVersion } from "gatsby-core-utils"
+import { gte, prerelease } from "semver"
 
 const typePrefix = `Contentful`
 const makeTypeName = type => _.upperFirst(_.camelCase(`${typePrefix} ${type}`))
 
+const GATSBY_VERSION_MANIFEST_V2 = `4.3.0`
+const gatsbyVersion = getGatsbyVersion()
+const gatsbyVersionIsPrerelease = prerelease(gatsbyVersion)
+const shouldUpgradeGatsbyVersion =
+  gte(gatsbyVersion, GATSBY_VERSION_MANIFEST_V2) && !gatsbyVersionIsPrerelease
```

Comment:
```suggestion
  lt(gatsbyVersion, GATSBY_VERSION_MANIFEST_V2) && !gatsbyVersionIsPrerelease
```

Reply:
Should this be "less than" instead of "greater than or equal"? i.e. if the current `gatsbyVersion` is less than `GATSBY_VERSION_MANIFEST_V2`, then `shouldUpgradeGatsbyVersion` is true.

Reply:
You're totally right, good catch 😅